docs/encyclopedia/application-message-passing.mdx

```diff
@@ -168,6 +170,8 @@ The start call will give you a handle you can use to track the Update, determine
 
 If you want to send an Update to another Workflow such as a Child Workflow from within a Workflow, you should do so within an Activity and use the Temporal Client as normal.
 
+Use [Update Validators](#update-validators) and [Update IDs](#exactly-once-message-processing) to stay within the system limits in both [Cloud](/cloud/limits#per-workflow-execution-update-limits) and [Self-Hosted](/self-hosted-guide/defaults).
```

Comment:
@drewhoskins-temporal @dandavison  what do you think about this?

Reply:
Agreed; nice addition.